jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/layout/HiddenTests.java

```diff
@@ -24,21 +26,24 @@
     public void before() {
         hiddenPage.open();
         hiddenPage.isOpened();
-    }
+        WebDriverFactory.getDriver().manage().window().maximize();
 
-    @Test
```

Comment:
Why you delete this annotation ? We don't need to verify element attributes in @BeforeMethod section

Reply:
Fixed